tests/foreman/api/test_new_repository.py

```diff
@@ -0,0 +1,112 @@
+"""Unit tests for the new ``repositories`` paths.
+
+:Requirement: Repository
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Repositories
+
+:Assignee: chiggins
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from requests.exceptions import HTTPError
+
+from robottelo.api.utils import enable_rhrepo_and_fetchid
+from robottelo.config import settings
+from robottelo.constants import PRDS
+from robottelo.constants import REPOS
+from robottelo.constants import REPOSET
+
+
+@pytest.mark.skip_if_open('BZ:2137950')
+@pytest.mark.tier1
+def test_negative_disable_repository_with_cv(
+    module_org, module_entitlement_manifest_org, target_sat
```

Comment:
Non-blocking, but it's not necessary to use both `module_org` and `module_entitlement_manifest_org`. Currently, `module_entitlement_manifest_org` inherits `module_org`, so passing both to the test is a bit redundant.

Reply:
Also, module_entitlement_manifest_org is returning module_org. So agree with @synkd !